src/urllib3/util/retry.py

```diff
@@ -27,6 +28,49 @@
 )
 
 
+# TODO: In v2 we can remove this sentinel and metaclass with deprecated options.
```

Comment:
Would be handy to have somekind of "TODO: Remove in v2: ..." fixed comment string we can search for later.

Reply:
I figure `v2` is enough since any mention of v2 in pre-v2 should be suspect. I'll do a big removal PR once we cut a `1.x` branch